app/meals/page.js

```diff
@@ -0,0 +1,63 @@
+"use client"
+
+import Navbar from "../shared_components/navbar";
+import MealSection from "../shared_components/meal_section";
+import styled from "styled-components";
+// import data from "./constant";
+import {useState, useEffect} from 'react';
+
+
+const StyledWrapperDiv = styled.div`
+  display: flex;
+  width: 100%;
+  background: #FCF8F3;
+  justify-content: center;
+
+  .content {
+    max-width: 1350px;
+  }
+
+  h1 {
+    font-size: 30pt;
+    font-optical-sizing: auto;
+    font-weight: 600;
+    font-style: normal;
+    margin: 30px;
+  }
+
+
+`
+
+
+export default function Meals() {
+    
+  const [mealsData, setMealsData] = useState([]);
+
+
+  useEffect(() => {
+    fetch('http://localhost:3001/meals').then(res => {
```

Comment:
about having `http://localhost:3001` hardcoded across the app. IT's better to have some variable, like DOMAIN_URL=http://localhost:3001. Then export this value where you need it. 
As a result, it will be easier to deploy to production in the future. There won't be locahost anymore, and you will change it only on one place instead of many.